robotd/devices.py

```diff
@@ -128,28 +130,43 @@ def status(self):
         return {'temperature': temp_milli_degrees / 1000}
 
 
+class NoZoneFound(ValueError):
+    """Indicates that a search for a zone file failed."""
+    pass
+
+
 class GameState(Board):
     """
     State storage for the game, keeps a store of everything it has received.
     """
 
+    FILE_GLOB = '/media/usb?/zone-?'
+    FILE_REGEX = re.compile('zone-(\d)')
```

Comment:
Why is there a separate match (glob + regex) for these? Surely a single one is enough?

Reply:
The glob is here to find the files, the regex is here to extract the number. We could regex over the entire filesystem, but that doesn't seem very efficient (my assumption is that glob has some understanding of the filesystem when it searches).

Reply:
Do you think naming the latter `ZONE_REGEX` clarify this sufficiently?